docs/README.md

```diff
@@ -1,101 +1,20 @@
+# Robyn's documentation
 
-[![Twitter](https://badgen.net/badge/icon/twitter?icon=twitter&label)](https://twitter.com/robyn_oss)
-[![Gitter](https://badges.gitter.im/robyn_/community.svg)](https://gitter.im/robyn_/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-[![Downloads](https://static.pepy.tech/personalized-badge/robyn?period=total&units=international_system&left_color=grey&right_color=blue&left_text=Downloads)](https://pepy.tech/project/robyn)
-[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
-[![GitHub tag](https://img.shields.io/github/tag/sansyrox/robyn?include_prereleases=&sort=semver&color=black)](https://github.com/sansyrox/robyn/releases/)
-[![License](https://img.shields.io/badge/License-BSD_2.0-black)](#license)
-[![Discord](https://img.shields.io/discord/999782964143603713?label=discord&logo=discord&logoColor=white&style=for-the-badge&color=blue)](https://discord.gg/qKF5sSnC)
+## 🔎 Overview
 
-[![view - Documentation](https://img.shields.io/badge/view-Documentation-blue?style=for-the-badge)](https://sansyrox.github.io/robyn/#/)
+Welcome to Robyn's documentation!
 
-Robyn is an async Python backend server with a runtime written in Rust, btw.
+Robyn is a Python web framework coupled with an async server implementation written in Rust. You can find the source code on [Github](https://github.com/sansyrox/robyn), contributions are very welcome!
 
-## 📦 Installation
+On this website, you will find more information on how to use Robyn.
 
-You can simply use Pip for installation.
+If you do not find what you were looking for here or have some suggestions to improve the docs, feel free to open an issue on the [Github repo](https://github.com/sansyrox/robyn), or ask for help on the official [Discord server](https://discord.gg/rkERZ5eNU8).
 
-```
-pip install robyn
-```
+## ✨ Special thanks
 
-Or, with [conda-forge](https://conda-forge.org/)
-
-```
-conda install -c conda-forge robyn
-```
-
-## 🤔 Usage
-
-```python
-from robyn import Robyn
-
-app = Robyn(__file__)
-
-
-@app.get("/")
-async def h(request):
-    return "Hello, world!"
-
-app.start(port=8080)
-```
-
-## 💡 Features
-
-- Under active development!
-- Written in Rust, btw xD
-- A multithreaded Runtime
-- Extensible
-- A simple API
-- Sync and Async Function Support
-- Dynamic URL Routing
-- Multi Core Scaling
-- WebSockets!
-- Middlewares
-- Hot Reloading
-- Community First and truly FOSS!
-
-## 🗒️ Contributor Guidelines
-
-Feel free to open an issue for any clarification or for any suggestions.
-
-If you're feeling curious. You can take a look at a more detailed architecture [here](https://github.com/sansyrox/robyn/blob/main/docs/architecture.md).
-
-## ⚙️ To Develop Locally
-
-1. Install the development dependencies: `pip install -r dev-requirements.txt`
-
-1. Install the pre-commit git hooks: `pre-commit install`
-
-1. Add more routes in the `integration_tests/base_routes.py` file(if you like).
```

Comment:
@AntoineRR , I think we should not delete this. If people will want to have a quick start documentation summary, they will have to go to the previous page. But if we keep it here, they won't have to move away from the documentation.

I believe that we need a proper custom documentation website revamp but I think we can keep this for now.

What say? @AntoineRR

Reply:
Ok that makes sense, let's keep this readme as a copy of the one on the repo for now then. We'll see later how to deal with this properly.